templates/node/src/client.ts.twig

```diff
@@ -201,7 +201,9 @@ class Client {
     }
 
     async chunkedUpload(method: string, url: URL, headers: Headers = {}, originalPayload: Payload = {}, onProgress: (progress: UploadProgress) => void) {
-        const file = Object.values(originalPayload).find((value) => value instanceof File);
+        const fileIndex = Object.values(originalPayload).findIndex((value) => value instanceof File);
+        const fileParam = Object.keys(originalPayload)[fileIndex];
+        const file = originalPayload[fileParam];
```

Comment:
```suggestion
        const [fileParam, file] = Object.entries(originalPayload).find(
          ([, value]) => value instanceof File
        );
```

I'd have done something like this, feel free to get inspired or just ignore :-P

Reply:
Used, improved with `?? []` for scenario if there isnt any file param